text.cabal

```diff
@@ -170,6 +170,8 @@ library
     Data.Text.Internal.Search
     Data.Text.Internal.Unsafe
     Data.Text.Internal.Unsafe.Char
+    Data.Text.Internal.Validate
+    Data.Text.Internal.Validate.Simd
```

Comment:
Please make this module conditional: `if flag(simdutf) exposed-modules: Data.Text.Internal.Validate.Simd`

Reply:
Will do. Also, I'll remove the conditionals from the `Simd` module itself since it's compilation will be guarded in this way.